samples/client/petstore/cpprest/api/UserApi.h

```diff
@@ -45,63 +47,81 @@ class  UserApi
     /// This can only be done by the logged in user.
     /// </remarks>
     /// <param name="body">Created user object</param>
-    pplx::task<void> createUser(std::shared_ptr<User> body);
+    pplx::task<void> createUser(
+        std::shared_ptr<User> body
+    );
     /// <summary>
     /// Creates list of users with given input array
     /// </summary>
     /// <remarks>
     /// 
     /// </remarks>
     /// <param name="body">List of user object</param>
-    pplx::task<void> createUsersWithArrayInput(std::vector<std::shared_ptr<User>> body);
+    pplx::task<void> createUsersWithArrayInput(
+        std::vector<std::shared_ptr<User>> body
+    );
     /// <summary>
     /// Creates list of users with given input array
     /// </summary>
     /// <remarks>
     /// 
     /// </remarks>
     /// <param name="body">List of user object</param>
-    pplx::task<void> createUsersWithListInput(std::vector<std::shared_ptr<User>> body);
+    pplx::task<void> createUsersWithListInput(
+        std::vector<std::shared_ptr<User>> body
+    );
     /// <summary>
     /// Delete user
     /// </summary>
     /// <remarks>
     /// This can only be done by the logged in user.
     /// </remarks>
     /// <param name="username">The name that needs to be deleted</param>
-    pplx::task<void> deleteUser(utility::string_t username);
+    pplx::task<void> deleteUser(
+        utility::string_t username
+    );
     /// <summary>
     /// Get user by user name
     /// </summary>
     /// <remarks>
     /// 
     /// </remarks>
     /// <param name="username">The name that needs to be fetched. Use user1 for testing. </param>
-    pplx::task<std::shared_ptr<User>> getUserByName(utility::string_t username);
+    pplx::task<std::shared_ptr<User>> getUserByName(
+        utility::string_t username
+    );
     /// <summary>
     /// Logs user into the system
     /// </summary>
     /// <remarks>
     /// 
     /// </remarks>
-    /// <param name="username">The user name for login</param>/// <param name="password">The password for login in clear text</param>
-    pplx::task<utility::string_t> loginUser(utility::string_t username, utility::string_t password);
+    /// <param name="username">The user name for login</param>
+    /// <param name="password">The password for login in clear text</param>
+    pplx::task<utility::string_t> loginUser(
+        utility::string_t username,
+        utility::string_t password
+    );
     /// <summary>
     /// Logs out current logged in user session
     /// </summary>
     /// <remarks>
     /// 
     /// </remarks>
-    
-    pplx::task<void> logoutUser();
+    pplx::task<void> logoutUser(
+    );
```

Comment:
This looks wack.

Reply:
Unfortunately it can't be avoided because of how mustache templates work..
I'd rather have this than all arguments on the same line, it can get quite lengthy and unreadable with several optional parameters.